examples/wails/Packager.toml

```
@@ -0,0 +1,71 @@
+name = "wails-example"
+before-packaging-command = "wails build"
```

```suggestion
before-packaging-command = "wails build -noPackage"
```

i forgot that i don't have access to this repo, but the change is small enough for a suggestion.